lib/pipeline/webpackProcess.js

```diff
@@ -38,26 +35,44 @@ class WebpackProcess extends ProcessWrapper {
     });
   }
 
-
-  webpackRun(filename, options, includeModules, importsList, detectErrors, callback) {
+  webpackRun(filename, options, includeModules, importsList, detectErrors, realCwd, callback) {
+    const self = this;
     let defaultOptions = {
+      mode: self.env === 'development' ? 'none' : 'production',
```

Comment:
Should keep it to development for now. We mostly want prod only when using `build` or a flag. Otherwise, we'll end up using prod using testnet or even privatenet

Reply:
When I tried it, `development` mode output from webpack wasn't working correctly, I need to take a closer look. What's happening in this line is that if embark's mode is `development` then webpack's optimization mode gets set to `none`, which is basically a dev build without some fancy webpack dev extras (those would be nice to have, but require more investigation). Only when embark's mode is `production` will webpack's mode flip to `production`.

Reply:
Change made.